### ISSUE TITLE
Clicking wiki button passes through to minimap

### ISSUE THREAD

**Issue by CreativeTechGuy** [runelite/runelite#16999]
### Type

Incorrect behavior

### Operating System

Windows

### OS Version

Windows 10

### Bug description

In fixed mode: If you click the part of the Wiki icon that is overlapping the map (top left corner), it'll both perform the wiki button's functionality, but also move your character since it counts as a click of the minimap behind it. This overlapping area is reasonably large (maybe 5-10% of the banner's clickable area).

Note: This only happens with the RuneLite Wiki plugin, if I disable the plugin and use the built-in one instead it works as expected.

### Expected behavior

Clicking on a button would consume the click event and not let the event trigger functionality behind it. (1 click = 1 action)

### RuneLite version

RuneLite: 1.10.11.1
Launcher: 2.6.4

### Logs

N/A

https://github.com/runelite/runelite/assets/12002072/af52d021-2581-4c3f-b71f-3a34aa833e41

**Comment by CreativeTechGuy** [comment]
@Nightfirecat Since I cannot reply on the Development channel in Discord, I wanted to clarify this issue is not with the Jagex Wiki button but the RuneLite one. I've added more details to the post and added a video for clarity.

PS: And I'd say 5-10% of the time where a click does something extra you didn't intend is pretty significant haha

**Comment by abextm** [comment]
This happens on vanilla, we don't change the minimap's clickmask.

**Comment by CreativeTechGuy** [comment]
@abextm I don't think that is true. Here's a full video showing that I am using the RuneLite Wiki lookup first, and the issue is present. Then I disable the RuneLite version and enable the Jagex one and the issue cannot be reproduced. I am not suggesting that the minimap's clickmask is different. But the functionality of the wiki button is inarguably different as hopefully the video shows. Let me know if there's something else I can do to prove this, but I don't think this issue should be closed.

https://github.com/runelite/runelite/assets/12002072/432abb5c-6349-4599-b655-601ab9a30eef

**Comment by abextm** [comment]
I can reproduce it on vanilla, therefore it is a vanilla bug.

**Comment by CreativeTechGuy** [comment]
Can you please explain what is happening in my video demonstration then? Somehow one works and the other doesn't, both in RuneLite.

**Comment by abextm** [comment]
OK, there are 2 clickmasks for the fixed mode minimap, varying on if the vanilla wiki button is on. (1200 `fixed_map_clickmask`, 2422 `fixed_map_clickmask_wiki`) The one for the wiki button on is very tight, but the one without has no mask over the button at all. This is what you are experiencing. This does not happen in resizable, which has 3 masks - `resize_map_mask` 1178 for when data orbs are disabled entirely, `resize_map_clickmask` 2154 for when data orbs are on, and `resize_map_clickmask_bond` 3513 for when data orbs and the bond orb are on

**Comment by CreativeTechGuy** [comment]
Oh wonderful! I'm not crazy. Phew!

I cannot seem to find those masks/values anywhere in the RuneLite source, is this one of those things that is in the closed-source portion of the codebase? I'm happy to try to fix it if you wouldn't mind giving me some pointers. Although given your reply, I suspect you might have already figured out the solution and could do it faster than it'd take to teach. 😅
